manifests/init.pp

```diff
@@ -1,27 +1,54 @@
-# Class: bareos
+# @summary This puppet module configures and manage all aspects of an complex bareos installation
 #
-# Authors
-# -------
+# @author Richard Hillmann <richie@project0.de>
 #
-# Richard Hillmann <richie@project0.de>
+# @param repo_release
```

Comment:
all those new puppet-strings comments and datatypes create a huge diff. Do you think it's possible to cherry-pick those and provide them in a dedicated PR?

Reply:
took them out